test/mocks/mockMiniMemos.js

```diff
@@ -4,6 +4,7 @@ const mockedMiniMemosPdfAndWeb = {
       3: {
         _id: 'EI1220_20182_3',
         courseCode: 'EI1220',
+        applicationCodes: ['3'],
         changedBy: 'krisa',
         // koppsRoundId: '3',
         ladokRoundIds: ['3'],
```

Comment:
`ladokRoundIds` has to be removed, in order to be sure tests work

Reply:
ladokRoundIds was removed from everywhere